src/init.cpp

```diff
@@ -188,7 +188,7 @@ void Shutdown()
     if (!lockShutdown)
         return;
 
-    /// Note: Shutdown() must be able to handle cases in which AppInit2() failed part of the way,
+    /// Note: Shutdown() must be able to handle cases in which AppInitMain() failed part of the way,
```

Comment:
let's just remove mention of a function here and call it "initialization". It may be any of the initialization-related functions that failed.